.github/workflows/pypi.yaml

```diff
@@ -0,0 +1,43 @@
+name: Publish pydoctor to PyPI on tags
```

Comment:
I think that this file should be removed.

Reply:
Sorry that was an error